src/module.ts

```diff
@@ -25,13 +28,26 @@ export default defineNuxtModule<ModuleOptions>({
     siteKey: nuxt.options.dev ? '1x00000000000000000000AA' : undefined,
     addValidateEndpoint: false,
   }),
-  async setup (options, nuxt) {
+  async setup(options, nuxt) {
     const siteKey = options.siteKey || nuxt.options.runtimeConfig.public?.turnstile?.siteKey
     if (!siteKey) {
       console.warn('`@nuxtjs/turnstile` is disabled as no site key was provided.')
       return
     }
 
+    const secretKeyPath =
+      options.secretKeyPath || nuxt.options.runtimeConfig.turnstile?.secretKeyPath
+    let secretKeyFromPath
+    if (secretKeyPath) {
+      secretKeyFromPath = fs.existsSync(secretKeyPath)
+        ? fs.readFileSync(secretKeyPath, 'utf-8')
+        : undefined
+    }
+    if (!secretKeyFromPath) {
+      console.warn('The secret key from the path could not be found.')
+    }
+    options.secretKey = secretKeyFromPath
```

Comment:
Also, this would always set the `secretKey` to `undefined` if nothing is read from file, I think.

Reply:
I wasn't sure how to handle a wrong key input. One solution would be to handle it like a wrong site key, return a warning and 'disable' the turnstile module

Reply:
I mean that using a `secretKey` only as it's currently supported would not work anymore as it appears to me.

Reply:
When a secret key is passed in the nuxt config then it is used. The secret key from the config has priority over the key from the file

Reply:
I think Jonas' point is that you would need to do this instead or you are unsetting `options.secretKey` when it exists:

```suggestion
    options.secretKey = secretKeyFromPath || options.secretKey
```

Reply:
I think this change can be reverted as the logic changed how `secretKeyPath` is treated.